third_party/2and3/click/utils.pyi

```diff
@@ -72,7 +72,7 @@ class KeepOpenFile:
 
 
 def echo(
-    message: Optional[Union[bytes, Text]] = ...,
+    message: Optional[object] = ...,
```

Comment:
You can also remove the `Optional` (which is short for a Union with None).

Reply:
Yeah -- somehow I missed that.  Thanks.